plugins/github-issues/src/hooks/useGetIssuesBeRepoFromGitHub.test.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const mockGraphQLQuery = jest.fn(() => ({}));
+jest.mock('./useOctokitGraphQL', () => ({
+  useOctokitGraphQL: jest.fn(() => mockGraphQLQuery),
+}));
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import { useGetIssuesByRepoFromGitHub } from './useGetIssuesByRepoFromGitHub';
+
+describe('useGetIssuesBeRepoFromGitHub', () => {
+  it('should call GitHub API with correct query with fragment for each repo', async () => {
+    const Helper = () => {
+      const getIssues = useGetIssuesByRepoFromGitHub();
+
+      getIssues(['mrwolny/yo-yo', 'mrwolny/yoyo', 'mrwolny/yo.yo'], 10);
+
+      return <div />;
+    };
+
+    render(<Helper />);
+
+    expect(mockGraphQLQuery).toHaveBeenCalledTimes(1);
+    expect(mockGraphQLQuery).toHaveBeenCalledWith(
+      '\n' +
```

Comment:
Can we do this as a multiline string using backticks instead of the old string concat method?

Reply:
Might look a little cleaner

Reply:
Thanks for the comment @benjdlambert, I agree it would probably make it nicer, but it's not that easy to achieve. The GraphQL queries require adequate whitespace in appropriate places and the current test syntax makes it much easier to test the query in that context. 

I gave it a try, failed and personally, I don't think it's worth it.